docs/user/dql/functions.rst

```diff
@@ -859,6 +859,29 @@ Example::
     +------------------------------------------------+----------------------------------+------------------------------------------------+
 
 
+CONVERT_TZ
+----------
+
+Description
+>>>>>>>>>>>
+
+Usage: convert_tz(datetime, from_timezone, to_timezone) constructs a local datetime converted from the from_timezone to the to_timezone.
+
+Argument type: DATETIME, STRING, STRING
+
+Return type: DATETIME
+
+Example::
+
+  os> SELECT CONVERT_TZ("2010-10-10 10:10:10", "+01:00", "-10:00"), CONVERT_TZ("test", "+01:00", "-10:00"), CONVERT_TZ("2010-10-10 10:10:10", "test", "-10:00"), CONVERT_TZ("2010-10-10 10:10:10", "+01:00", "-14:00"), CONVERT_TZ("2010-10-10 10:10:10", "+00:00", "+00:00")
```

Comment:
Could you split each expression into separate examples? Also if you alias the expression, the result will be easier to understand.